packages/buckets/src/buckets.ts

```diff
@@ -303,10 +347,37 @@ export class Buckets extends GrpcAuthentication {
    * }
    * ```
    */
+  async getOrCreate(name: string, options?: GetOrCreateOptions): Promise<{ root?: RootObject; threadID?: string }> 
```

Comment:
Probably should create a type for this return here at some point. Not a blocker though.